<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>898</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="SimpleCardWidget" name="SimpleCardWidget_2">
     <property name="minimumSize">
      <size>
       <width>780</width>
       <height>90</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>100</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <widget class="PrimaryPushButton" name="import_model_btn">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>导入模型</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="PrimaryPushButton" name="set_word_label_btn">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>飞浆设置</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="RadioButton" name="yolo_cls_radio_btn">
            <property name="text">
             <string>自定义yolo-cls</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="RadioButton" name="panddle_onnx_radio_btn">
            <property name="text">
             <string>自定义paddle-onnx</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="VerticalSeparator" name="VerticalSeparator"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="RadioButton" name="panddle_radio_btn">
            <property name="text">
             <string>panddle_ocr</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="RadioButton" name="clip_radio_btn">
            <property name="text">
             <string>clip</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="ToggleButton" name="auto_tbtn">
            <property name="text">
             <string>全自动模式</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="ToggleButton" name="half_auto_tbtn">
            <property name="text">
             <string>半自动模式</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="VerticalSeparator" name="VerticalSeparator_3"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="BodyLabel" name="Confidence_label">
          <property name="text">
           <string>置信度: 0</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="BodyLabel" name="Identification_label">
            <property name="text">
             <string>识别标签:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="LineEdit" name="text_label_edit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="VerticalSeparator" name="VerticalSeparator_4"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <item>
           <widget class="PrimaryPushButton" name="load_model_btn">
            <property name="text">
             <string>加载模型</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="ToggleButton" name="start_predict_btn">
            <property name="text">
             <string>开启识别</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="PrimaryPushButton" name="impor_dataset_btn">
            <property name="text">
             <string>导入数据集</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="PrimaryPushButton" name="load_clip_labels_btn">
            <property name="maximumSize">
             <size>
              <width>96</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>加载clip标签</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="SimpleCardWidget" name="SimpleCardWidget">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>300</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="CardWidget" name="CardWidget_2">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>0</height>
             </size>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <property name="spacing">
              <number>5</number>
             </property>
             <property name="leftMargin">
              <number>5</number>
             </property>
             <item>
              <widget class="PixmapLabel" name="PixmapLabel"/>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="CardWidget" name="CardWidget">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>10000</width>
              <height>50</height>
             </size>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_10">
             <item>
              <widget class="PrimaryPushButton" name="skip_btn">
               <property name="maximumSize">
                <size>
                 <width>200</width>
                 <height>50</height>
                </size>
               </property>
               <property name="text">
                <string>跳过</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="PrimaryPushButton" name="save_btn">
               <property name="maximumSize">
                <size>
                 <width>200</width>
                 <height>50</height>
                </size>
               </property>
               <property name="text">
                <string>保存</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="ListWidget" name="image_path_list">
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PushButton</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>PrimaryPushButton</class>
   <extends>PushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>RadioButton</class>
   <extends>QRadioButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>ToggleButton</class>
   <extends>PushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>PixmapLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>VerticalSeparator</class>
   <extends>QWidget</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>CardWidget</class>
   <extends>QFrame</extends>
   <header>qfluentwidgets</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SimpleCardWidget</class>
   <extends>CardWidget</extends>
   <header>qfluentwidgets</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BodyLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>ListWidget</class>
   <extends>QListWidget</extends>
   <header>qfluentwidgets</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
